templates/web.template.yml

```diff
@@ -12,6 +13,7 @@ params:
 run:
   - exec: /usr/local/bin/ruby -e 'if ENV["HOSTNAME"] == "intercity.example.com"; puts "Aborting! Domain is not configured!"; exit 1; end'
   - exec: /usr/local/bin/ruby -e 'if ENV["SECRET_KEY_BASE"] == "secret_key_base"; puts "Aborting! Secret key base is not configured!"; exit 1; end'
+  - exec: /usr/local/bin/ruby -e 'if ENV["FROM_EMAIL"] == "example@example.com"; puts "Aborting! FROM_EMAIL is not configured"; exit 1; end'
```

Comment:
One thing that comes to mind is that this won't `exit` when one of these ENV vars is not present at all, right?

Maybe also do a `nil` check?
```ruby
# something like
if ENV["FROM_EMAIL"].in?(nil, "example@example.com")
# or
if !ENV["FROM_EMAIL"] || ENV["FROM_EMAIL"] == "example@example.com"
```
Or is it very unlikely that these ENV vars won't be present at all?